src/mocks/handlers.ts

```diff
@@ -0,0 +1,9 @@
+import { rest } from 'msw'
```

Comment:
I'd name this folder something like `testHelpers` coz `mock` refers to fake static values rather than the logic to generate ones. It's a bit subjective tho.